Makefile

```diff
@@ -162,3 +164,46 @@ ifneq ($(CLOUD_PROVIDER),)
 else
 	$(error CLOUD_PROVIDER is not set)
 endif
+
+### PODVM IMAGE BUILDING ###
+
+REGISTRY ?= quay.io/confidential-containers
+
+PODVM_DISTRO ?= ubuntu
+
+PODVM_BUILDER_IMAGE ?= $(REGISTRY)/podvm-builder-$(PODVM_DISTRO):$(VERSIONS_HASH)
+PODVM_BINARIES_IMAGE ?= $(REGISTRY)/podvm-binaries-$(ARCH):$(VERSIONS_HASH)
```

Comment:
Missing `$(PODVM_DISTRO)` in binaries image name.

Reply:
Are the binaries DISTRO specific?

Reply:
If they are dynamically linked they are, yes.

Reply:
See https://github.com/confidential-containers/cloud-api-adaptor/issues/1093#issuecomment-1600991335 for an example.

Reply:
Ah interesting, thanks. I'll make the name distro specific then